src/HotChocolate/Data/src/Data/Filters/Convention/FilterConventionDefinition.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace HotChocolate.Data.Filters
+{
+    public class FilterConventionDefinition
+    {
+        public IEnumerable<FilterOperationConventionDefinition> Operations { get; set; } =
```

Comment:
Also here I see 2 styles, which are we gonna choose:

A)

```csharp
public IEnumerable<FilterOperationConventionDefinition> Operations { get; set; } =
    Enumerable.Empty<FilterOperationConventionDefinition>();
```

B)

```csharp
public IEnumerable<FilterOperationConventionDefinition> Operations { get; set; } 
    = Enumerable.Empty<FilterOperationConventionDefinition>();
```

@rstaib @PascalSenn

Reply:
A)

Reply:
A)